.drone.yml

```diff
@@ -442,21 +444,24 @@ steps:
     pull: if-not-exists
     image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind:latest
     environment:
-      DOCKER_PASSWORD:
-        from_secret: DOCKER_PASSWORD
-      DOCKER_REGISTRY_URL:
-        from_secret: DOCKER_REGISTRY_URL
-      DOCKER_USERNAME:
-        from_secret: DOCKER_USERNAME
+      AWS_ACCESS_KEY_ID:
+        from_secret: DEV_EGAR_PUBLIC_SITE_UI_AWS_ACCESS_KEY_ID
+      AWS_SECRET_ACCESS_KEY:
+        from_secret: DEV_EGAR_PUBLIC_SITE_UI_AWS_SECRET_ACCESS_KEY
```

Comment:
Question: Is the cron job possible to test or not?
* ideally should be tested, but can be left.

Reply:
1. - Tested successfully:
https://drone-gh.acp.homeoffice.gov.uk/UKHomeOffice/egar-public-site-ui/4241
2.- Please re-review the PR as I have added a few scripts to simplify maintainability